### PR TITLE
retrieve a complete url in http mode

### PR DESCRIPTION
**Pull request by rockem**

### CODE REVIEW

File: src/main/java/io/ticktok/server/auth/APISecurityConfiguration.java [hunk 1]
```diff
@@ -22,6 +22,9 @@ public APISecurityConfiguration(@Value("${http.auth-token}") String authToken) {
     @Override
     protected void configure(HttpSecurity httpSecurity) throws Exception {
         httpSecurity.
+                authorizeRequests().
```
Comment (amaziagur) [inline src/main/java/io/ticktok/server/auth/APISecurityConfiguration.java]: we need to add CORS support...

Comment (rockem) [reply src/main/java/io/ticktok/server/auth/APISecurityConfiguration.java]: can you elaborate?

Comment (amaziagur) [reply src/main/java/io/ticktok/server/auth/APISecurityConfiguration.java]: clients will get hit error for Cross-Origin resource sharing if you in the server will not enable it.
you need to configure what allowed origins and what methods and headers and such you enable.

https://spring.io/guides/gs/rest-service-cors/

Comment (rockem) [reply src/main/java/io/ticktok/server/auth/APISecurityConfiguration.java]: I think only browser client are affected by this, no?

Comment (amaziagur) [reply src/main/java/io/ticktok/server/auth/APISecurityConfiguration.java]: yes you are right. 
we have one.. from what i know.

Comment (rockem) [reply src/main/java/io/ticktok/server/auth/APISecurityConfiguration.java]: yea, but it shouldn't access this url 
this url is for clients usage only

**Pull request merged**